src/EA.Iws.Web/Areas/AdminExportNotificationMovements/Controllers/CaptureMovementController.cs

```diff
@@ -31,7 +31,6 @@ public CaptureMovementController(IMediator mediator)
         public async Task<ActionResult> Create(Guid id)
         {
             var model = new CreateViewModel();            
-            model.LatestCurrentMovementNumber = await mediator.SendAsync(new GetLatestMovementNumber(id));
```

Comment:
You can delete the LatestCurrentMovementNumber from the ViewModel as it is no longer required. Similarly the GetLatestMovementNumber request / handler are probably not used any more either so can be safely deleted.